packages/macros/src/macros-config.ts

```diff
@@ -60,7 +60,7 @@ export default class MacrosConfig {
 
   private internalSetConfig(fromPath: string, packageName: string | undefined, config: unknown) {
     if (this.cachedUserConfigs) {
-      throw new Error(`attempted to set config after we have already emitted our config`);
+      // throw new Error(`attempted to set config after we have already emitted our config` + fromPath);
```

Comment:
Just seeing if tests pass if this assertion is removed. I don't intend this to be part of the fix.